catalog/core/catalog-core-migratable/src/test/java/org/codice/ddf/catalog/migratable/impl/MigrationFileWriterTest.java

```diff
@@ -0,0 +1,215 @@
+
+import java.io.BufferedOutputStream;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.ObjectOutputStream;
+import java.io.OutputStream;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import org.apache.commons.io.FileUtils;
+import org.codice.ddf.migration.MigrationException;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.ArgumentCaptor;
+import org.mockito.runners.MockitoJUnitRunner;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.rule.PowerMockRule;
+
+import ddf.catalog.data.Result;
+import ddf.catalog.data.impl.MetacardImpl;
+import ddf.catalog.data.impl.ResultImpl;
+
+//@RunWith(PowerMockRunner.class)
+@RunWith(MockitoJUnitRunner.class)
+@PrepareForTest({MigrationFileWriter.class, File.class, FileUtils.class})
+public class MigrationFileWriterTest {
+
+    private static final int RESULT_COUNT = 5;
+
+    private static final Path DDF_BASE_PATH = Paths.get("ddf");
+
+    @Rule
+    public PowerMockRule rule = new PowerMockRule();
+
+    //@Mock
+    private Path mockPath;
+
+    //@Mock
+    private File mockFile;
+
+    //@Mock
+    private FileOutputStream mockFileOutputStream;
+
+    //@Mock
+    private BufferedOutputStream mockBufferedOutputStream;
+
+    //@Mock
+    private ObjectOutputStream mockObjectOutputStream;
+
+    private MigrationFileWriter fileWriter;
+
+    @Before
+    public void setup() throws Exception {
+        fileWriter = new MigrationFileWriter() {
+            @Override
+            ObjectOutputStream createObjectStream(OutputStream source) {
+                return mockObjectOutputStream;
+            }
+
+            @Override
+            BufferedOutputStream createBufferedStream(OutputStream source) {
+                return mockBufferedOutputStream;
+            }
+
+            @Override
+            FileOutputStream createFileStream(File file) {
+                return mockFileOutputStream;
+            }
+        };
+
+        mockPath = mock(Path.class);
+        mockFile = mock(File.class);
+        mockFileOutputStream = mock(FileOutputStream.class);
+        mockBufferedOutputStream = mock(BufferedOutputStream.class);
+        mockObjectOutputStream = mock(ObjectOutputStream.class);
+
+        when(mockPath.toFile()).thenReturn(mockFile);
+    }
+
+    private List<Result> loadList() {
+        final List<Result> results = new ArrayList<>();
+        for (int i = 0; i < RESULT_COUNT; i++) {
+            MetacardImpl metacard = new MetacardImpl();
+            metacard.setId("id" + String.valueOf(i));
+            results.add(new ResultImpl(metacard));
+        }
+        return results;
+    }
+
+    @Test
+    public void testInitCallsForceMkDir() throws Exception {
+        mockStatic(FileUtils.class);
```

Comment:
I'm not clear why you need to use Mockito here and not just let the export path be created and clean it up in a teardown method?